optimum/onnxruntime/trainer.py

```diff
@@ -528,7 +528,8 @@ def _inner_training_loop(
         model = ORTModule(self.model)
         self.model_wrapped = model
 
-        if args.deepspeed:
+        # Note transformers/trainer.py renamed this arg to is_deepspeed_enabled
+        if args.deepspeed: 
```

Comment:
In the latest `Trainer`, `args.deepspeed` is replaced by the attribute `self.is_deepspeed_enabled` fetched from the accelerator state.

Reply:
Changed